sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -83,28 +84,50 @@ public Schemas(Settings settings,
 
     public DocTableInfo getDroppableTable(TableIdent tableIdent) {
         TableInfo tableInfo = getTableInfo(tableIdent);
-        if (!(tableInfo instanceof DocTableInfo)) {
-            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
-                "The table %s is not dropable.", tableInfo.ident()));
-        }
+        Operation.blockedRaiseException(tableInfo, Operation.DROP);
         DocTableInfo docTableInfo = (DocTableInfo) tableInfo;
         if (docTableInfo.isAlias() && !docTableInfo.isPartitioned() && !isOrphanedAlias(docTableInfo)) {
             throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
                 "%s is an alias and hence not dropable.", tableInfo.ident()));
         }
+        if (docTableInfo.isClosed()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is closed. Write, Drop or Alter operations (other than OPEN) are not supported",
+                tableInfo.ident()));
+        }
         return docTableInfo;
     }
 
     public DocTableInfo getWritableTable(TableIdent tableIdent) {
         TableInfo tableInfo = getTableInfo(tableIdent);
+        if (Operation.isReadOnly(tableInfo)) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is read-only. Only READ operations are supported.", tableInfo.ident()));
+        }
+        DocTableInfo docTableInfo = (DocTableInfo) tableInfo;
+        if (docTableInfo.isAlias() && !docTableInfo.isPartitioned()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "%s is an alias. Only READ operations are supported.", tableInfo.ident()));
+        }
+        if (docTableInfo.isClosed()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is closed. Write, Drop or Alter operations (other than OPEN) are not supported",
```

Comment:
pls change to "...is closed. No operation beside opening it using ALTER is supported"